src/platform_impl/linux/x11/event_processor.rs

```diff
@@ -1316,19 +1297,6 @@ impl EventProcessor {
             let id = xev.detail as u32;
             let location = PhysicalPosition::new(xev.event_x, xev.event_y);
 
-            // Mouse cursor position changes when touch events are received.
-            // Only the first concurrently active touch ID moves the mouse cursor.
-            if is_first_touch(&mut self.first_touch, &mut self.num_touch, id, phase) {
-                let event = Event::WindowEvent {
-                    window_id,
-                    event: WindowEvent::CursorMoved {
-                        device_id: mkdid(util::VIRTUAL_CORE_POINTER),
-                        position: location.cast(),
-                    },
-                };
-                callback(&self.target, event);
-            }
-
```

Comment:
I think the point was that the touch moves the mouse pointer? In general, I'd _hold_ it a bit until pointer overhual since it won't break things due to things being _generic_.

Right now I'm not sure, so I'd suggest to research why this code is here in the first place, since it's clearly fixing some _not working_ thing.

Reply:
Its this https://github.com/rust-windowing/winit/issues/1261.
Which has a good use-case as well.

Will remove the removal of this part.